conjugate_map/conjCalcFunctions.py

```diff
@@ -1,35 +1,22 @@
 # functions for computing geomagnetic conjugate points.
 
 # Importing packages
-import numpy as np
-import pandas as pd
-import plotly.express as px
-import json
-import ogr
-import geopandas as gpd
-import aacgmv2
 import datetime as dt
-import numpy as np
 import os
-import plotly.graph_objects as go
-
-import pandas as pd
-smag = __import__('supermag-api')          # SuperMAG python API
-logon = 'kd8oxt'                              # SuperMAG ID
-
-import gpxpy
-import gpxpy.gpx
-
-import geopandas
 
-# converting from geographic to GSM
+import aacgmv2
 from geopack import geopack as gp 
 from geopack import t89
+import gpxpy
+import gpxpy.gpx
+import numpy as np
+import pandas as pd
+import plotly.express as px
+import plotly.graph_objects as go
 
-import datetime
 ############################################################################################################################### 
 
-def findconj(lat, lon, ut = dt.datetime.now(), is_verbose = 0, method = 'aacgm'):
+def findconj(lat, lon, ut = dt.datetime.utcnow(), is_verbose = 0, method = 'aacgm', limit = 60):
```

Comment:
`dt.datetime.utcnow` has been deprecated: https://discuss.python.org/t/deprecating-utcnow-and-utcfromtimestamp/26221.  Also, `is_verbose` should be a boolean (False/True).